src/app/shared/chips/models/chpis-item.model.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { ChipsItem, ChipsItemIcon } from './chips-item.model';
```

Comment:
This Spec file has a misspelling in the file name `chpis-item.model.spec.ts`

Reply:
fixed